precompiles/parachain-staking/src/lib.rs

```diff
@@ -88,6 +88,23 @@ where
 		Ok(points)
 	}
 
+	#[precompile::public("awardedPoints(uint256,address)")]
+	#[precompile::view]
+	fn awarded_points(
+		handle: &mut impl PrecompileHandle,
+		round: SolidityConvert<U256, u32>,
```

Comment:
> "`SolidityConvert` is not needed anymore since the new proc macro `#[precompile_utils::precompile]` already expand the right convertion code." ~ [librelois](https://github.com/PureStake/moonbeam/pull/1885#discussion_r1067970273)

So could just make it `round: u32` and change the Solidity input type to `uint32`.

Reply:
@librelois is this true? I updated the [code](https://github.com/PureStake/moonbeam/pull/2056/commits/407ef733ab0fe69d65ba0c3dc9429e68379fb677) and then the typescript tests started failing

Reply:
Yes I just checked again and I maintain that the use of `SolidityConvert` is useless for unsigned integers